src/testE2e.js

```diff
@@ -1,142 +0,0 @@
-
-const del = require('del');
-const gulp = require('gulp');
-const gulpBabel = require('gulp-babel');
-const gulpConnect = require('gulp-connect');
-const gulpNightwatch = require('gulp-nightwatch');
```

Comment:
At least this dependency can be removed from _package.json_ - maybe more?

Reply:
Removed.